changelog/fragments/confg-3-alpha-removed.yaml

```diff
@@ -0,0 +1,30 @@
+entries:
+  - description: >
+      PROJECT config version 3-alpha has been upgraded to version 3
+    kind: change
+    breaking: true
+    migration:
+      header: PROJECT config version 3-alpha must be upgraded to 3
+      body: >
+        Version 3-alpha has been removed after version 3 was released. While this change
+        is not technically breaking because the spec at that version was alpha, it was
+        used by default in `operator-sdk` commands so should be marked as breaking
+        and have a convenient migration path. The `config3alphato3` command will convert
+        most of your PROJECT file from version 3-alpha to 3, and leave comments with
+        directions where automatic conversion is not possible:
+
+        ```console
+        $ cat PROJECT
+        version: 3-alpha
+        resources:
+        - crdVersion: v1
+        ...
+        $ operator-sdk config3alphato3
+        Your PROJECT config file has been converted from version 3-alpha to 3. Please make sure all config data is correct.
+        $ cat PROJECT
+        version: "3"
+        resources:
+        - api:
+            crdVersion: v1
+        ...
+        ```
```

Comment:
```suggestion
        ```
         **NOTE** For more information about the project file attributes see the documentation [Migration from project config version "2" to "3"](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/migration/manually_migration_guide_v2_v3.md#migration-from-project-config-version-2-to-3)
```


Also, I think we need to link the migration guide in the upstream docs as well.  E.g

Reply:
Done.